src/components/Editor/SessionList.vue

```diff
@@ -22,17 +22,20 @@
 
 <template>
 	<NcPopover class="session-list" placement="bottom">
-		<div slot="trigger">
-			<button :title="label"
-				:aria-label="label"
-				class="avatar-list">
-				<div class="avatardiv icon-group" />
-				<AvatarWrapper v-for="session in sessionsVisible"
-					:key="session.id"
-					:session="session"
-					:size="40" />
-			</button>
-		</div>
+		<template #trigger="{ attrs }">
+			<div>
+				<button :title="label"
+					:aria-label="label"
+					class="avatar-list"
+					v-bind="attrs">
+					<div class="avatardiv icon-group" />
+					<AvatarWrapper v-for="session in sessionsVisible"
+						:key="session.id"
+						:session="session"
+						:size="40" />
```

Comment:
The div here should be switched to span and the divs inside AvatarWrapper also need to be span to not have block-level elements inside the inline button

Otherwise good!

Reply:
Thanks. I'll do it in another PR with fixing HTML validation issues.